### DISCUSSION TITLE
rooting T phone pro (REVVL 6 pro 5G)

### DISCUSSION THREAD

**Discussion by Hertrix147** [community/community#147113] [Mobile]
### Select Topic Area

Question

### Body

im struggling with rooting this device i used mtk client to back all partitions and then install magisk and patch boot.img to flash patched boot.img. i unlocked bootloader on phone with MTK Client (at least it tell me that it is unlocked, but flashing patched boot.img makes boot stuck on Telekom logo, would appreciate any help :c

**Comment by Akash1134** [comment]
Hi @Hertrix147 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
